src/index.html

```diff
@@ -11,6 +11,23 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#" method="post">
+      <label class="search__field">
+        <input
+          type="text"
+          class="input__field large--search__field"
+          data-qa="big"
+          placeholder="Try “Los Angeles”"
+        >
+      </label>
+      <label class="search__field">
+        <input
+          type="text"
+          class="input__field small--search__field"
+          data-qa="small"
+          placeholder="Try “Los Angeles”"
+        >
+      </label>
+    </form>
```

Comment:
You should have two forms one form per each input field. And add to big form tag data-qa="big" and to small form data-qa="small", in input inside big form you should use data-qa="keypress"